src/main/webapp/WEB-INF/view/admin.jsp

```diff
@@ -15,7 +15,7 @@
     <a id="navTitle" href="/">CodeU Chat App</a>
     <% if(request.getSession().getAttribute("user") != null){ %>
```

Comment:
Nitpicky: space between if and condition and { ?
if (request.getSession().getAttribute("user") != null) {

I feel like it looks less cleaner